### ISSUE TITLE
Attachment is not received

### ISSUE THREAD

**Issue by bhavinwebdesk** [devin-la/ionic-video-chat-v2-issues#115]
Hello @arzynik,

Steps to reproduce:
1) Login 1st user from Browser (Chrome Version 67.0.3396.99 (Official Build) (64-bit))
2) Login 2nd user from Android (v7.0.1. LG)
3) Send and attach Photo from one user to another in any way either 1st to 2nd or 2nd to 1st.
4) Message received but without attachment and not showing attachment.
5) When the receiver goes back to contact and reload chat, it is showing attachment.

**Expected Result:**
The attachment should be also shown when both user online and in-chat screen.

Should I change any code?

**Comment by therealjimwolff** [comment]
You probably should fix it and change the code and share with us all how you did it! I haven't gotten the chat to work yet. I did however get the video call between two working. What's the trick to get chat working between two users? Also does it support group video do you know?

**Comment by jackwebdesksolution** [comment]
Hay @therealjimwolff For chat it is working out of box. I set up as instructed in [here](https://github.com/devin-la/ionic-video-chat-support/wiki/Server). I don't know about group video as I am not able to start it.

**Comment by therealjimwolff** [comment]
Yeah followed all out of box instructions, funny I got the hardest part video call working but not chat. Maybe it's a platform thing, I'm using iOS and Macs. I'm sure I'll figure it out eventually. I was just wondering if there is a user interface trick I'm missing, because the "chat" button didn't do anything for me. I'll look at the logs, I know its my fault if others have it working no problem for chat.

**Comment by jackwebdesksolution** [comment]
I am able to do it with simple two users. In mac, one user logged in the safari while other one is in chrome(or other browser). For check issue I run my node.js in local and check log. Sometime it happen that Mongodb is not setup correctly.
